test/cucumber/ParkinSpaceSuggestion.feature

```diff
@@ -0,0 +1,48 @@
+Feature: parking space suggestion
+  As driver entering a parking lot
+  I want to find the best parking space
+  So that I can park my car
+
+  Scenario: setting user position
+      Given user “u” is active on the system
+        And the system doesn’t have stored the current parking that the user is in
+       When the user sets the parking lot
+       Then the system stores the user parking lot at that moment
+
+  Scenario: parking space search 
+      Given user “u” is active on the system
+        And the current parking is known
```

Comment:
Eu não entendi muito bem essa frase do cenário. A ideia não é que o usuário "x" está chegando no estacionamento e quer procurar uma vaga, ai a vaga "3" está livre e o sistema irá informar a vaga "3"? Porque "the current parking is know"? Acredito que justamente não tem "estacionamento atual", afinal o usuário não está estacionado.